src/System.Sorting/tests/SortingTest.cs

```diff
@@ -0,0 +1,84 @@
+            int[] array = new int[6000000];
+            int[] arrayTry = new int[2500];
+            for (int i = 0; i < 2500; i++)
+            {
+                arrayTry[i] = i;
+            }
+            for (int j = 0; j < 2400; j++)
+            {               
+                arrayTry.CopyTo(array, (j * 2500));
+            }
+           
+            int[] arrayClone = (int[])(array.Clone());
+
+
+            TimSort<int> timsort = new TimSort<int>(array, iComparer);
```

Comment:
Did you look at the existing tests marked with `[Performance]`?  I think it would be better to write these tests in that framework.  That way as the framework becomes more powerful (measuring GC Debt, etc.) you tests will get all the new features "for free".